### DISCUSSION TITLE
posts/2023/08/new-life-old-mac-linux/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#52] [Announcements]
# posts/2023/08/new-life-old-mac-linux/

A combination of Fedora, Proton, and the inevitable tinkering process produces amazing results.

https://www.brycewray.com/posts/2023/08/new-life-old-mac-linux/

**Comment by soundslikeorange** [comment]
Thanks you VERY much! :)

**Reply by brycewray** [reply]
@soundslikeorange Glad if it helped!
